ui/src/App.tsx

```diff
@@ -62,6 +62,13 @@ const App = () => {
         icon={OrganizationIcon}
         recordRepresentation={(record) => `${record.name}`}
       />
+      <Resource
+        name="organizations"
+        list={OrganizationList}
+        show={OrganizationShow}
+        icon={OrganizationIcon}
+        recordRepresentation={(record) => `${record.name}`}
+      />
```

Comment:
This is a duplicate of the `Resource` above it. Or am I missing something?

Reply:
Doh!